platforms/core-runtime/gradle-cli-main/src/main/java/org/gradle/launcher/GradleMain.java

```diff
@@ -28,6 +28,6 @@ public static void main(String[] args) throws Exception {
             System.exit(1);
         }
 
-        ProcessBootstrap.run("org.gradle.launcher.Main", args);
+        ProcessBootstrap.run("gradle-gradle-cli", "org.gradle.launcher.Main", args);
```

Comment:
🤔 That's a weird name. Should we instead call the subproject `:cli`, so this becomes `gradle-cli`?

Reply:
We already have `cli` and that deals with generic utilities that interact with the cli. This module implements the CLI frontend for the gradle client (the `gradle` CLI command). 

We prefix all module names with `gradle` automatically -- this is mostly invisible to us, but it is visible here. This is presumably to deal with name collisions at the module level, where modules lose their `group`. Thus, if some other library had some `cli` library, we ensure we don't collide with them.

This is actually a widespread problem that we don't really have a good solution for. Say you depend `org.foo:utilities:1.0` and `org.bar:utilities:1.0`. They will both alias down to `utilities-1.0.jar` on the classpath. I believe at the point of creating the Jar, we will emit a warning but I'm not sure there is much you can do here. 

Many organizations (and us included) automatically prefix their project names with their organization. We do this relatively seamlessly, but at my prior company, everything was "org.my-org.submodule:submodule-project-name" to avoid name collisions like these, as it was very common that each submodule would define some project called "utilities" that would otherwise alisas with other submodule's utilities. 

Sorry for all the words. My point is we should fix this somehow. But for this PR, I think this is fine.